corehq/apps/userreports/indicators/specs.py

```diff
@@ -97,3 +97,18 @@ class ChoiceListIndicatorSpec(PropertyReferenceIndicatorSpecBase):
 
     def get_operator(self):
         return IN_MULTISELECT if self.select_style == 'multiple' else EQUAL
+
+
+class LedgerBalancesIndicatorSpec(IndicatorSpecBase):
+    type = TypeProperty('ledger_balances')
+    product_codes = ListProperty(required=True)
+    ledger_section = StringProperty()
+    case_id_expression = DictProperty()
+
+    def get_case_id_expression(self):
+        from corehq.apps.userreports.expressions.factory import ExpressionFactory
+        expression_spec = self.case_id_expression or {
+            "type": "property_name",
+            "property_name": "_id",
+        }
+        return ExpressionFactory.from_spec(expression_spec)
```

Comment:
I figured since this will most likely always be this same expression, it makes sense to not require you to specify.  Happy to hear arguments if you disagree.